developers/slashing.md

```diff
@@ -0,0 +1,36 @@
+---
+description: This section covers the slashing mechanics for validators in Celestia.
+---
+
+# Slashing on Celestia
+
+Slashing is mechanism employed in PoS blockchains that is used to deter
+and punish malicious behavior. It functions by removing a percentage
+of a validators stake each time they act harmfully towards the network.
+
+Celestia is built with the Cosmos-SDK and implements the module `x/slashing`.
+
+If a Validator gets slashed, delegators bonded to that validator will also
+have the same percentage of their delegated funds slashed.
+
+The following are the conditions for a validator to get slashed:
+
+1. **Downtime**: If a validator is offline for more than 75% of a rolling window
+   of the last 5,000 blocks, they will lose 0% of their stake.
```

Comment:
[blocking] the way this is phrased is incorrect. If a validator doesn't sign 75% of the last 5,000 blocks they will be jailed. So if they are offline for more than 25% of that rolling window, they will be jailed.